### PR TITLE
PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component

### PR DESCRIPTION
**Pull request by ThereseRoca**
# Issue Link

[PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component](https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-12)

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1691147307571499

# Definition of Done
Create a card component for the listings containing:
- [x] Image
- [x] Label
- [x] Ratings
- [x] Reviews
- [x] Price
- [x] Type
- [x] Capacity

# Pre-conditions 
none

# Expected Output
image

# Screenshots
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-
-# Editor directories and files
-.idea
-.vscode
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.sw?
+.DS_Store
```
Comment (roseaugusto) [inline .gitignore]: what was change here?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (roseaugusto) [inline package-lock.json]: why is this change? there is no new package being installed so I think no need for this one to be modified

Comment (ThereseRoca) [reply package-lock.json]: It just seems to be there po ma'am even if I don't stage it for commit during that time.  It also occurred on some of the PR of my co-interns ma'am.

File: .gitignore (hunk 1 above)
Comment (ThereseRoca) [reply .gitignore]: I accidentally put the login page before ma'am in .gitignore maam so I removed it there ma'am.

File: package-lock.json (hunk 2 above)
Comment (LennardUy104) [reply package-lock.json]: Removed duplicate package-lock.json

**Comment by LennardUy104** [comment]
Resolve conflict and clean up code

**Review by roseaugusto — APPROVED** [review]

**Pull request merged**